packages/application-components/src/components/internals/page-header-title.tsx

```diff
@@ -7,30 +7,34 @@ import styled from '@emotion/styled';
 type Props = {
   title: string;
   titleSize: 'big' | 'small';
+  truncate: boolean;
   subtitle?: string | ReactElement;
   children?: never;
 };
-const defaultProps: Pick<Props, 'titleSize'> = {
+const defaultProps: Pick<Props, 'titleSize' | 'truncate'> = {
   titleSize: 'small',
+  truncate: true,
```

Comment:
The default value should be `false`. So when you do need to use it, you can simply pass it as `truncate`. It's the same in the `<Text>` components.

Of course then we need to explicitly set it in the other components that expect it.